docs/multi-stage-query/reference.md

```diff
@@ -575,14 +575,14 @@ The following table lists query limits:
 
 | Limit | Value | Error if exceeded |
```

Comment:
In this table, since many of them reference the table just below, let's take out the links entirely. For example, line 586 doesn't have a link on the error code.